python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/test_assets.py

```diff
@@ -84,3 +84,47 @@ def test_can_build_two_assets():
 
         assert defs.get_assets_def("asset1")
         assert defs.get_assets_def("asset2")
+
+
+def test_sling_modes():
```

Comment:
Recommend that you parameterize `test_build_sling_asset` with `mode` and `expected_row_counts` after a sling invocation.

Reply:
that make a lot more sense